src/api_manager/auth/lib/json_util.cc

```diff
@@ -57,6 +61,44 @@ const char *GetPropertyValue(const grpc_json *json, const char *key,
   return nullptr;
 }
 
+const std::string GetPrimitiveFieldValue(const std::string &json,
```

Comment:
Are you using "" to indicate invalid?   can we use bool for that?

Reply:
Done.